src/libraries/Microsoft.PowerFx.Interpreter/Functions/Library.cs

```diff
@@ -10,6 +10,7 @@
 using Microsoft.PowerFx.Core.Texl;
 using Microsoft.PowerFx.Core.Types;
 using Microsoft.PowerFx.Core.Utils;
+using Microsoft.PowerFx.Interpreter;
```

Comment:
Not needed anymore. Same for other files.

Reply:
Fixed